openhands/server/routes/files.py

```diff
@@ -1,4 +1,5 @@
 import os
+from typing import Any, Dict, List, Optional, Union
```

Comment:
We should use built typing instead

- `dict`
- `list`
- `| None` instead of `Optional`
- `|` instead of `Union`